src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SerializerTest.php

```diff
@@ -78,6 +79,15 @@ public function testSerializeTranslatableBackedEnum()
 
         $this->assertEquals('GET', $serializer->serialize(TranslatableBackedEnum::Get, 'yaml'));
     }
+
+    public function testSerializeJsonSerializableBackedEnum()
```

Comment:
Seeing how the BackedEnumNormalizer has a test itself, wouldn't it be a more fitting place to test the new behavior in the normalizers [BackedEnumNormalizerTest](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Serializer/Tests/Normalizer/BackedEnumNormalizerTest.php#L25)?